internal/armada/server/lease.go

```diff
@@ -954,11 +954,11 @@ func (q *AggregatedQueueServer) addAvoidNodeAffinity(
 		}
 
 		changed := addAvoidNodeAffinity(jobs[0], labels, func(jobsToValidate []*api.Job) error {
-			if ok, err := validateJobsCanBeScheduled(jobsToValidate, allClusterSchedulingInfo); !ok {
+			if ok, responseItems, err := validateJobsCanBeScheduled(jobsToValidate, allClusterSchedulingInfo); !ok {
 				if err != nil {
-					return errors.WithMessage(err, "can't schedule at least 1 job")
+					return errors.WithMessagef(err, "can't schedule %d job(s)", len(responseItems))
```

Comment:
What does the error say here if you e.g. submit a batch of 100 jobs and 99 of them are fine but 1 can be scheduled.  Does it say "can't schedule 100 job(s)" or "can't schedule 1 jobs(s)"?

Reply:
Good question - I've changed it so hopefully it's more clear - the code is now
```
return errors.WithMessagef(err, "can't schedule %d (out of %d submitted) job(s)", len(responseItems), len(jobsToValidate))
```